src/Extension/Engine/SiteTreePublishingEngine.php

```diff
@@ -84,82 +86,187 @@ public function getToDelete()
     /**
      * @param array $toUpdate
      * @return $this
+     * @deprecated 6.0.0 use {@link setUrlsToUpdate()} This method is still used internally for backwards compatability,
```

Comment:
```suggestion
     * @deprecated 6.0.0 Use setUrlsToUpdate() instead.
```
Deprecation messages are formatted in a standard way as per https://docs.silverstripe.org/en/contributing/release_process/#deprecating-api
Applies to the other deprecations as well.